ui/src/common/QualityInfo.js

```diff
@@ -33,12 +34,14 @@ export const QualityInfo = ({ record, size, className }) => {
   }
 
   return (
-    <Chip
-      className={clsx(classes.chip, className)}
-      variant="outlined"
-      size={size}
-      label={info}
-    />
+    <Fade in={songTitleHover} timeout={500}>
```

Comment:
The component should not incorporate Fade, as it is not its concern. For example, now in the SongGrid and AlbumSongs, the quality does not appear at all! Please move this logic to the AudioTitle component